src/test/java/tests/CustomersTest.java

```diff
@@ -0,0 +1,80 @@
+package tests;
+
+import library.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.*;
+
+
+
+
+public class CustomersTest extends TestBase {
+    @Test
+    public void addNewCustomer() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.CUSTOMERS);
+        CustomersPage customersPage = new CustomersPage(driver,null);
+        customersPage.saveNewCustomer("Amazon");
+        customersPage.saveNewCustomer("TATA Elxsi");
+        Assert.assertTrue(customersPage.match);
```

Comment:
Assert should pass only if the customer you have added is present on the table. Recheck the logic for that